src/result.js

```diff
@@ -113,10 +115,20 @@ class Result {
 }
 
 function insertIntoIndexHTML(html, head, body) {
-  html = html.replace("<!-- EMBER_CLI_FASTBOOT_BODY -->", body);
+  const headComment = "<!-- EMBER_CLI_FASTBOOT_HEAD -->";
+  const bodyComment = "<!-- EMBER_CLI_FASTBOOT_BODY -->";
+
+  const warnMissing = missing => {
```

Comment:
Rename this function to something better since you aren't warning now but throwing an error.